lib/dockerize_rails/docker_commands/docker_build.rb

```diff
@@ -0,0 +1,67 @@
+module DockerizeRails
+  module DockerCommands
+    module DockerBuild
+      def self.build_rails
+        build_docker_image(
+          "#{Constants::CONFIG_DIRECTORY_NAME}/#{Constants::RAILS_DIRECTORY_NAME}/Dockerfile",
+          "#{DRConfig.application_name}_mysql"
+        )
+        0
+      rescue Docker::Error::NotFoundError => e
+        puts
+        puts e.to_s.red
+        puts
+        1
+      end
+
+      def self.build_mysql
+        if DRConfig.linked_database? && DRConfig.databases[DRConfig.application_env] == 'mysql'
+          build_docker_image(
+            "#{Constants::CONFIG_DIRECTORY_NAME}/#{Constants::MYSQL_DIRECTORY_NAME}/Dockerfile",
+            "#{DRConfig.application_name}_mysql"
+          )
+        end
+        0
+      rescue Docker::Error::NotFoundError => e
+        puts
+        puts e.to_s.red
+        puts
+        1
+      end
+
+      def self.build_postgres
+        if DRConfig.linked_database? && DRConfig.databases[DRConfig.application_env] == 'postgresql'
+          build_docker_image(
+            "#{Constants::CONFIG_DIRECTORY_NAME}/#{Constants::PG_DIRECTORY_NAME}/Dockerfile",
+            "#{DRConfig.application_name}_postgres"
+          )
+        end
+        0
+      rescue Docker::Error::NotFoundError => e
```

Comment:
__[reek]__
* [UncommunicativeVariableName] has the variable name 'e' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)

<!-- Comment by [SideCI](https://sideci.com) -->